Commands/ConfigTrait.php

```diff
@@ -72,6 +73,7 @@ protected function loadConfig(InputInterface $input)
         $config['app-env'] = $this->optionOrConfigValue($input, 'app-env', $config);
         $config['debug'] = $this->optionOrConfigValue($input, 'debug', $config);
         $config['logging'] = $this->optionOrConfigValue($input, 'logging', $config);
+        $config['static'] = $this->optionOrConfigValue($input, 'static', $config);
```

Comment:
Needs type casting: `$config['static'] = (boolean)$this->optionOrConfigValue($input, 'static', $config);`